guides/common/modules/con_using-boot-disks-to-provision-hosts.adoc

```diff
@@ -1,4 +1,10 @@
 [id="using-boot-disks-to-provision-hosts"]
 = Using boot disks to provision hosts
 
-You can use boot disks to boot hosts without PXE firmware.
+Some hardware does not provide a PXE boot interface.
```

Comment:
This is correct, but it's very hardware-centric; most users work with virtual machines as well.
When you operate with VMs, you don't have hardware problem, more likely you don't have network that does not support network boot.

I would add `or your environment is not supporting network boot`.
(Note: Not all images work in unmanaged DHCP networks)